docs/core/docker/build-container.md

```diff
@@ -14,6 +14,7 @@ This tutorial teaches you how to build a Docker image that contains your .NET Co
 You'll learn to:
 
 > [!div class="checklist"]
```

Comment:
These are the ones it would be good if someone could check on the review.docs.microsoft.com side

Reply:
looks fine!